config/deployments/env_specific.toml

```diff
@@ -0,0 +1,211 @@
+# like card details
+[locker]
+host = ""                                                             # Locker host
+host_rs = ""                                                          # Rust Locker host
+mock_locker = true                                                    # Emulate a locker locally using Postgres
+locker_signing_key_id = "1"                                           # Key_id to sign basilisk hs locker
+locker_enabled = true                                                 # Boolean to enable or disable saving cards in locker
+redis_temp_locker_encryption_key = "redis_temp_locker_encryption_key" # Encryption key for redis temp locker
+
+[log.console]
+enabled = true
+level = "DEBUG"
+log_format = "json"
+
+[log.file]
+enabled = true
+level = "DEBUG"
+log_format = "json"
+
+# Telemetry configuration for metrics and traces
+[log.telemetry]
+traces_enabled = false                                # boolean [true or false], whether traces are enabled
+metrics_enabled = false                               # boolean [true or false], whether metrics are enabled
+ignore_errors = false                                 # boolean [true or false], whether to ignore errors during traces or metrics pipeline setup
+sampling_rate = 0.1                                   # decimal rate between 0.0 - 1.0
+otel_exporter_otlp_endpoint = "http://localhost:4317" # endpoint to send metrics and traces to, can include port number
+otel_exporter_otlp_timeout = 5000                     # timeout (in milliseconds) for sending metrics and traces
+use_xray_generator = false                            # Set this to true for AWS X-ray compatible traces
+route_to_trace = ["*/confirm"]
+
+[lock_settings]
+delay_between_retries_in_milliseconds = 500 # Delay between retries in milliseconds
+redis_lock_expiry_seconds = 180             # Seconds before the redis lock expires
+
+# Main SQL data store credentials
+[master_database]
+username = "db_user"      # DB Username
+password = "db_pass"      # DB Password. Use base-64 encoded kms encrypted value here when kms is enabled
+host = "localhost"        # DB Host
+port = 5432               # DB Port
+dbname = "hyperswitch_db" # Name of Database
+pool_size = 5             # Number of connections to keep open
+connection_timeout = 10   # Timeout for database connection in seconds
+queue_strategy = "Fifo"   # Add the queue strategy used by the database bb8 client
+
+[payment_link]
+sdk_url = "http://localhost:9090/0.16.7/v0/HyperLoader.js"
+
+[payment_method_auth]
+pm_auth_key = "pm_auth_key" # Payment method auth key used for authorization
+redis_expiry = 900          # Redis expiry time in milliseconds
+
+[proxy]
+http_url = "http://proxy_http_url"    # Outgoing proxy http URL used by SES for sending emails since SES isn't accessible inside the VPC
+https_url = "https://proxy_https_url" # Outgoing proxy https URL used by SES for sending emails since SES isn't accessible inside the VPC
```

Comment:
This isn't specific to SES exactly. The application also has provision to interact with services on the Internet via a proxy, and you can configure the URLs for the proxies.

The `http_url` and the `https_url` are the proxy URLs for HTTP and HTTPS traffic, respectively. In some cases, the value for both these URLs can be the same, if the same proxy is handling both HTTP and HTTPS traffic from the application.